src/box/lua/tuple.c

```diff
@@ -757,6 +757,34 @@ lbox_tuple_info(lua_State *L)
 	return 1;
 }
 
+/**
+ * Push to Lua stack a table with the information about a format of a tuple.
+ * Elements of a table are structs {'name', 'type'}.
+ */
+static int
+lbox_tuple_get_format(lua_State *L)
+{
+	int argc = lua_gettop(L);
+	if (argc != 1)
+		luaL_error(L, "Usage: tuple:format()");
+
+	struct tuple *tuple = luaT_checktuple(L, 1);
+	struct tuple_format *format = tuple_format(tuple);
+	if (format->data == NULL) {
+		lua_createtable(L, 0, 0);
+		return 1;
+	}
+
+	const char *data = format->data;
+	luamp_decode(L, luaL_msgpack_default, &data);
+	luaL_findtable(L, LUA_GLOBALSINDEX, "box.internal.space", 1);
+	lua_getfield(L, -1, "denormalize_format");
```

Comment:
At first glance, these strings should be constants.

Reply:
In this file these strings are used only once each. I don't see a necessity to wrap them into separate `char*` constants. Why do you think this change should be done?

Reply:
Twice already:
https://github.com/tarantool/tarantool/blob/c5b3e5949b43131276b492e59dec5a41fa49573a/src/box/lua/tuple_format.c#L122-L123

But with the `grep LUA_GLOBALSINDEX -r src` it seems like ok.